spec/support/supplier_with_access_token.rb

```diff
@@ -2,8 +2,7 @@
 
 RSpec.shared_context 'with supplier with access token' do
   let(:supplier) { create(:supplier) }
-  let(:application) { create(:application, owner_id: supplier.id) }
-  let(:access_token) { create(:access_token, application: application).token }
+  let(:access_token) { 'spoofed-token' }
```

Comment:
I'm wary about this change: this specific context ties an access token to a supplier. In some tests we are (or should be) testing that supplier1 cannot access supplier2's moves. I suggest keeping this context to test specific permissions cases, and updating most specs to use spoofed authenticaiton.

Reply:
For tests that do need to check we get the correct details back (essentially authorisation rather than authentication) I've kept the real access token, or in some cases added a specific context to use the real token. For speed it's important to only use the full authentication stack where we absolutely have to, so I'd prefer to keep this as the default. I did get failing specs around suppliers seeing moves they weren't supposed to, so the test coverage is good for that - and is one example where it's not using the spoofed token.

Reply:
ok, in that case how about renaming this context 'with supplier with spoofed access token': to make it clearer that the access token is _not_ tied to supplier (as previously it was)

Reply:
Sure 👍